runners/direct-java/src/main/java/org/apache/beam/runners/direct/ParDoMultiOverrideFactory.java

```diff
@@ -261,6 +262,12 @@ public PCollectionTuple expand(PCollection<? extends KeyedWorkItem<K, KV<K, Inpu
     public String getUrn() {
       return DIRECT_STATEFUL_PAR_DO_URN;
     }
+
+    @Override
+    public RunnerApi.FunctionSpec getSpec() {
+      throw new UnsupportedOperationException(
```

Comment:
this looks like `NotSerializable`

Reply:
It is, but `NotSerializable` is the payload translator base class, while this is the `RawPTransform`. It is a good point, and I've replaced all of this leveraging of `RawPTransformTranslator` with explicitly registered `NotSerializable` translators.